src/core/apiConfig.tsx

```diff
@@ -85,6 +85,12 @@ export interface ReservationRequestPayload {
     roomReservationsIds: string[],
     transportReservationsIds: string[],
     userId: string,
+
+    hotelName: string,
+    roomReservationsNames: string[],
+    locationNameFrom: string,
```

Comment:
tu też nowy konstrukt `locationName`, czy to sam region, region + kraj, czy też może sam kraj? nie wiadomo

Reply:
Okej, uwzględniłem to